packages/@aws-cdk/aws-rds/lib/props.ts

```diff
@@ -197,6 +205,34 @@ export abstract class Credentials {
    * @default - none
    */
   public abstract readonly secret?: secretsmanager.Secret;
+
+  /**
+   * The characters to exclude from the generated password.
+   * Only used if {@link generatePassword} if true.
+   *
+   * @default - the DatabaseSecret default exclude character set (" %+~`#$&*()|[]{}:;<>?!'/@\"\\")
+   */
+  public abstract readonly excludeCharacters?: string;
+}
+
+/**
+ * Options used in the {@link SnapshotCredentials.fromGeneratedPassword} method.
+ */
+export interface SnapshotCredentialsFromGeneratedPasswordOptions {
+  /**
+   * KMS encryption key to encrypt the generated secret.
+   *
+   * @default - default master key
+   */
+  readonly encryptionKey?: kms.IKey;
+
+  /**
+   * The characters to exclude from the generated password.
+   * Has no effect if {@link password} has been provided.
```

Comment:
This docstring doesn't work here. There's no password to provide if you're using these options.

Reply:
Yep, changed!